include/date/date.h

```diff
@@ -6879,7 +6879,7 @@ from_stream(std::basic_istream<CharT, Traits>& is, const CharT* fmt,
                 }
                 else  // !command
                 {
-                    if (isspace(*fmt))
+                    if (isspace(Traits::to_int_type(*fmt)))
```

Comment:
I don't think this one is fixed.  Perhaps `static_cast<unsigned char>(*fmt)`.  That isn't perfect, but it would at least avoid UB.

Reply:
Changed it to static_cast, which is actually what char_traits<char>::to_int_type does (at least in libstdc++ and msvc).